content/en/docs/how-tos/adding-a-cluster-profile.md

```diff
@@ -133,6 +133,15 @@ separate key-value pairs. You'll have two new key-value pairs, where the keys
 are `ssh-publickey` and `ssh-privatekey` and they store the file contents of
 the SSH public and private key, respectively.
 
+#### Supplying an AWS IP Pool
+
+It is possible to take advantage of AWS's [BYOIP feature](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html)
+to allow your test's ephemeral clusters to utilize configured IP Pools for cost savings. To utilize this, a few steps must be taken:
+1. Set up IP Pool in utilized region(s). See AWS [docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html). Note that this feature can only handle **one** IP Pool per region.
+2. Add `Boskos` configuration for _every_ region you intend to utilize. See the [example](https://github.com/openshift/release/blob/6a8f889b353fd3764b7c877bb0cd52cf7ea68aba/core-services/prow/02_config/_boskos.yaml#L435-L438), noting, the `name` should be `{some-unique-name}-{region}`.
```

Comment:
Instead of `{some-unique-name}-{region}` lets provide some standard naming convention eg `aws-{project-name}-ip-pools-{region}`

Reply:
hmm, that is an interesting point. It is entirely unnecessary from our end to have that, and we won't have any sort of enforcement. I suppose recommending it can't hurt. I will update that.